cli/internal/git/utils.go

```diff
@@ -232,3 +240,132 @@ func deleteBranchIfExists(gitDirectory string, branchName plumbing.ReferenceName
 
 	return nil
 }
+
+func CreateZarfOrg() error {
+	// Establish a git tunnel to send the repos
+	tunnel := k8s.NewZarfTunnel()
+	tunnel.Connect(k8s.ZarfGit, false)
+	defer tunnel.Close()
+
+	body := map[string]string{
+		"username":   config.ZarfGitOrg,
+		"visibility": "limited",
+	}
+	jsonData, err := json.Marshal(body)
+	if err != nil {
+		return err
+	}
+
+	request, err := netHttp.NewRequest("POST", fmt.Sprintf("http://%s:%d/api/v1/orgs", config.IPV4Localhost, k8s.PortGit), bytes.NewBuffer(jsonData))
```

Comment:
This file is inconsistent with how the request, response, body variables are named.

- [ ] make consistent